R/pkg/R/context.R

```diff
@@ -319,6 +319,27 @@ spark.addFile <- function(path, recursive = FALSE) {
   invisible(callJMethod(sc, "addFile", suppressWarnings(normalizePath(path)), recursive))
 }
 
+#' Adds a JAR dependency for Spark tasks to be executed in the future.
+#'
+#' The \code{path} passed can be either a local file, a file in HDFS (or other Hadoop-supported
+#' filesystems), an HTTP, HTTPS or FTP URI, or local:/path for a file on every worker node.
```

Comment:
is `local:/path` referring to windows drive/path, or the actual text `local:/` should be there?

Reply:
I think it refers the actual `local:/`:

https://github.com/apache/spark/blob/b2463fad718d25f564d62c50d587610de3d0c5bd/core/src/main/scala/org/apache/spark/SparkContext.scala#L1838